src/web_socket.toit

```diff
@@ -189,14 +208,28 @@ class WebSocket:
     current_reader_ = null
 
   /**
-  Abruptly closes the WebSocket.
-  Does not send a close packet first.
+  May abruptly close the WebSocket.
```

Comment:
?
```suggestion
  Closes the WebSocket.
  May close abruptly in an unclean way.
```
Would be nice to continue the second sentence with "[...] way. To avoid this ..."